src/bp/ui-shared-lite/utils/storage.ts

```diff
@@ -0,0 +1,64 @@
+// @ts-nocheck
+
+import Cookie from 'js-cookie'
```

Comment:
not a fan of having this duplicated between shared-lite and shared ... can you import this one in ui-shared ?

Reply:
Forgot to delete the file, it's not even referenced